File: test-projects/java-project/src/test/java/org/acme/GreetingResourceTest.java
package org.acme;

import acme.MyPackage;
import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@ExtendWith(IntelliJGradleQuarkusTestExtension.class)
@QuarkusTest
public class GreetingResourceTest {

    @Test
    public void testHelloEndpoint() {
        System.out.println(new MyPackage());
        given()
                .when().get("/hello")
                .then()
                .statusCode(200)
                .body(is("Hello RESTEasy"));
    }

}

File: test-projects/kotlin-project/src/test/kotlin/acme/GreetingResourceTest.kt
package acme

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.acme.IntelliJGradleQuarkusTestExtension
import org.hamcrest.Matchers.`is`
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith

@ExtendWith(IntelliJGradleQuarkusTestExtension::class)
@QuarkusTest
class GreetingResourceTest {
    @Test
    fun testHelloEndpoint() {
        println(MyPackage())
        given()
            .`when`().get("/hello")
            .then()
            .statusCode(200)
            .body(`is`("Hello RESTEasy 42"))
    }
}